omnibus/files/private-chef-cookbooks/private-chef/recipes/add_ons_wrapper.rb

```diff
@@ -48,14 +48,16 @@ def report
 
   require 'mixlib/install'
 
+  platform_info = Mixlib::Install.detect_platform
```

Comment:
Yeah basically `detect_platform` instance method sets the platform options on the `Mixlib::Install` instance.